### DISCUSSION TITLE
re AggregatedQueuesDataTiles

### DISCUSSION THREAD

**Discussion by crispinmOHQ** [twilio-professional-services/flex-project-template#617] [Q&A]
![image](https://github.com/user-attachments/assets/3aaa674f-a456-4470-8648-f51d0b57f486)
![image](https://github.com/user-attachments/assets/7ec786d4-eec8-4654-aaad-c71c64d6905b)
![image](https://github.com/user-attachments/assets/579bae47-3eb6-457e-bcbf-8f1149adf534)

I'm trying to modify the component inside the big box by removing it and replacing it with my own but I can't seem to find the right flex component, I've checked the component key but it doesn't work either. I also followed this [link](https://www.twilio.com/docs/flex/developer/ui/queues-view-programmability) as well but can't seem to remove the said component. Any ideas how to customised this?

**Comment by dremin** [comment]
Hi @crispinmOHQ! You can remove the "waiting" section by clicking the "Edit view" button, choosing the "Cards" tab, and then within the "All" section, uncheck "Waiting tasks".
